frontend/src/components/SeedlotRegistrationSteps/OwnershipStep/SingleOwnerInfo/styles.scss

```diff
@@ -0,0 +1,85 @@
+@use '../../../../theme/variables.scss' as vars;
+
+.single-owner-info-container {
+  padding: 1rem;
+  width: 100%;
+
+  .single-owner-info-col {
+    margin: 0;
+    padding-bottom: 2rem;
+
+    input {
+      background-color: var(--#{vars.$bcgov-prefix}-field-02);
+    }
+
+    button {
+      border: none;
+    }
+
+    button::before {
+      background-color: var(--#{vars.$bcgov-prefix}-field-02);
+    }
+
+    button::after {
+      background-color: var(--#{vars.$bcgov-prefix}-field-02);
+    }
+  }
+
+  .#{vars.$bcgov-prefix}--list-box__menu-item {
+    background-color: var(--#{vars.$bcgov-prefix}-layer-02);
+  }
+
+  .#{vars.$bcgov-prefix}--list-box__menu-item--highlighted {
+    background-color: var(--#{vars.$bcgov-prefix}-layer-hover-02);
+  }
+
+  .#{vars.$bcgov-prefix}--list-box__menu-item--active {
+    background-color: var(--#{vars.$bcgov-prefix}-layer-hover-02);
+  }
+
+  .#{vars.$bcgov-prefix}--label--disabled {
+    color: var(--#{vars.$bcgov-prefix}-text-secondary);
+  }
+
+  .#{vars.$bcgov-prefix}--form__helper-text--disabled {
+    color: var(--#{vars.$bcgov-prefix}-text-secondary);
+  }
+
+  .#{vars.$bcgov-prefix}--list-box--disabled {
+    input {
+      background-color: var(--#{vars.$bcgov-prefix}-field-01);
+      color: var(--#{vars.$bcgov-prefix}-text-primary);
+    }
+  }
+
+  .#{vars.$bcgov-prefix}--text-input:disabled {
+    border-bottom: 1px solid var(--#{vars.$bcgov-prefix}-border-subtle-00);
+    -webkit-text-fill-color: var(--#{vars.$bcgov-prefix}-text-primary);
+    background-color: var(--#{vars.$bcgov-prefix}-field-01);
+    color: var(--#{vars.$bcgov-prefix}-text-primary);
+  }
```

Comment:
yes these are needed since we have a grey layer in the background of the input, so the default carbon colour will get blended in with the grey background.  You'll see what I mean if you comment these out.